packages/next/build/flying-shuttle.ts

```diff
@@ -298,7 +296,9 @@ export class FlyingShuttle {
       const c = pageNames.length - u
       Log.info(`found ${c} changed and ${u} unchanged page${u > 1 ? 's' : ''}`)
     } else {
-      Log.warn(`flying shuttle had no pages we can reuse`)
+      Log.warn(
+        `the flying shuttle requires a full rebuild due to application changes`
```

Comment:
Requires or is going to do?

Reply:
I would interpret this as "run `next build` again"

Reply:
> flying shuttle is going to perform a full rebuild due to changes across all pages

👍?